diff --git a/afirma-simple/src/main/java/es/gob/afirma/standalone/SimpleAfirma.java b/afirma-simple/src/main/java/es/gob/afirma/standalone/SimpleAfirma.java
index ae05aa195..3144533cf 100644
--- a/afirma-simple/src/main/java/es/gob/afirma/standalone/SimpleAfirma.java
+++ b/afirma-simple/src/main/java/es/gob/afirma/standalone/SimpleAfirma.java
@@ -1053,9 +1053,6 @@ public final class SimpleAfirma implements PropertyChangeListener, WindowListene
 				LOGGER.info("Iniciando entorno grafico"); //$NON-NLS-1$
 				saf.initGUI(null, null);
 
-				LOGGER.info("Comprobando si es una version de Java compatible"); //$NON-NLS-1$
-				checkJavaVersion(saf.getMainFrame());
-
 				// Comprobamos si el almacen de confianza con el certificado SSL esta instalado correctamente
     			final CheckTrustKeyStoreTask checkTrustStoreTask = new CheckTrustKeyStoreTask(saf.getMainFrame());
     			checkTrustStoreTask.execute();
